shard-operator/src/main/java/com/redhat/service/bridge/shard/operator/ManagerSyncServiceImpl.java

```diff
@@ -46,7 +50,19 @@
     BridgeExecutorService bridgeExecutorService;
 
     @Inject
-    Tokens tokens;
+    OidcClient client;
+
+    Tokens currentTokens;
+
+    @PostConstruct
+    public void init() {
+        try {
+            currentTokens = client.getTokens().await().atMost(SSO_CONNECTION_TIMEOUT);
+        } catch (RuntimeException e) {
+            LOGGER.error("Fatal error: could not fetch initial authentication token from sso server.");
```

Comment:
@r00ta Probably useful to print the `Exception` here as part of the log message, otherwise we'll find it very hard to debug what's going wrong.